app/route_rangers_api/views.py

```diff
@@ -31,15 +33,29 @@ def dashboard(request, city: str):
     # get commute
 
     # get paths
-    routes = TransitRoute.objects.filter(city="CHI").values(
-        "geo_representation", "route_name", "color"
+    routes = TransitRoute.objects.filter(city=CITY_CONTEXT[city]["DB_Name"])  # .values(
+    # MultiLineString needs to be serialized into a GeoJson object for Leaflet to
+    # work with it.
+    # to serialize into GeoJson, need to get out entire Django model object, not just
+    # the .values("geo_representation", "route_name", "color")
+    # with .values() you get "AttributeError: 'dict' has no component 'meta'"
+    routes_json = serialize(
+        "geojson",
+        routes,
+        geometry_field="geo_representation",
+        fields=("route_name", "color"),
     )
 
     # stations
     stations = TransitStation.objects.values().filter(
         city=CITY_CONTEXT[city]["DB_Name"]
     )
-    lst_coords = [[point["location"].x, point["location"].y] for point in stations]
+    # print(stations)
```

Comment:
don't need print statements in production if I am reading this correctly  (line 58 as well)